libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsUserIdentity.java

```diff
@@ -2,15 +2,25 @@
 
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
-import uk.gov.di.ipv.core.library.domain.ReturnCode;
 
 import java.util.List;
 
 @ExcludeFromGeneratedCoverageReport
-public record AuditExtensionsUserIdentity(
-        @JsonProperty String levelOfConfidence,
-        @JsonProperty boolean ciFail,
-        @JsonProperty boolean hasMitigations,
-        @JsonInclude(JsonInclude.Include.NON_NULL) @JsonProperty List<ReturnCode> returnCode)
-        implements AuditExtensions {}
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AuditExtensionsUserIdentity<T> implements AuditExtensions {
+    @JsonProperty private String levelOfConfidence;
+    @JsonProperty boolean ciFail;
+    @JsonProperty boolean hasMitigations;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("https://vocab.account.gov.uk/v1/returnCode")
```

Comment:
Sorry I think this is wrong on the ticket, I don't think it's helpful to use this name in the audit event - we can just use `returnCode`.

Reply:
I've updated the ACs